matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/RoomDirectoryService.kt

```diff
@@ -40,4 +40,6 @@ interface RoomDirectoryService {
      * Set the visibility of a room in the directory
      */
     suspend fun setRoomDirectoryVisibility(roomId: String, roomDirectoryVisibility: RoomDirectoryVisibility)
+
+    suspend fun checkAliasAvailability(aliasLocalPart: String?) : Result<Unit>
```

Comment:
See my comment in the impl

Reply:
yes initialy wanted to return Either<RoomAliasError, Unit>, but the creation is strange